cli/commands/console.go

```diff
@@ -113,6 +122,26 @@ func (o *ConsoleOptions) Run() error {
 	templateProvider := console.NewDefaultTemplateProvider(basePath, templateFilename)
 	consoleTemplateVersion := templateProvider.GetTemplateVersion(o.EC.Version)
 	consoleAssetsVersion := templateProvider.GetAssetsVersion(o.EC.Version)
+
+	if o.CliExternalEndpoint == "" {
+		o.CliExternalEndpoint = "http://" + o.Address + ":" + o.APIPort
+	}
+	if o.ServerExternalEndpoint == "" {
+		o.ServerExternalEndpoint = o.EC.Config.ServerConfig.Endpoint
+	}
+
+	// FIXME: My Router struct
+	r := &cRouter{
+		g,
+		t,
+	}
```

Comment:
@scriptonist I'm marking 4 `FIXME` comments that I need help with. Basically wanted to know if there are alternate structures or methods that need to be used in order to get this PR up to speed.